tests/functional/test_patient_access.py

```diff
@@ -7,9 +7,6 @@
 #     generate_random_phone_number,
```

Comment:
can this be removed now?

Reply:
I left these as they are used by the drafted/commented-out patient access tests.